### PR TITLE
Add discord-formatted export support

### PR DESCRIPTION
**Pull request by pavelbraginskiy**

### CODE REVIEW

**Comment by pavelbraginskiy** [comment]
MegaMek component of MegaMek/megameklab#1517

File: megamek/src/megamek/common/util/DiscordExportUtil.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package megamek.common.util;
+
```
Comment (SJuliez) [inline megamek/src/megamek/common/util/DiscordExportUtil.java:2]: Please add the copyright from 
https://github.com/MegaMek/megamek/wiki/MegaMek-Coding-Style-Guide

File: megamek/src/megamek/common/MechView.java [hunk 2]
```diff
@@ -44,6 +45,11 @@
  * @since January 20, 2003
  */
 public class MechView {
+    public static enum ViewFormatting {
```
Comment (SJuliez) [inline megamek/src/megamek/common/MechView.java:48]: This aren't big deals, but I'd encourage everyone to give classes their own file. Our existing classes are all too long. Also, as constants, the enum values are typically written in capitals (DISCORD...)

**Review by SJuliez — APPROVED** [review]
This looks great. I have some rather minor comments. I'll approve this and still be happy if you could address them (changing the enums willl affect the MML PR too).

**Comment by pavelbraginskiy** [comment]
@SJuliez Done!

**Pull request merged**